app/src/main/java/cm/aptoide/pt/account/view/LoginSignUpFragment.java

```diff
@@ -154,7 +147,11 @@ public void setupToolbarDetails(Toolbar toolbar) {
   }
 
   @Override public int getContentViewId() {
-    return R.layout.fragment_login_sign_up;
+    if (navigateToHome) {
```

Comment:
While navigate to home variable is at this moment a reliable source in identifying the context (wizard or not) it should not be used to decide which layout to show. In the future if we need to change the navigateToHome logic we will have the side effect of possibly inflating the wrong xml.